packages/connectivity/connectivity/pubspec.yaml

```diff
@@ -12,12 +12,15 @@ flutter:
         pluginClass: ConnectivityPlugin
       ios:
         pluginClass: FLTConnectivityPlugin
+      macos:
+        default_package: connectivity_macos
 
 dependencies:
   flutter:
     sdk: flutter
   meta: "^1.0.5"
   connectivity_platform_interface: ^1.0.2
+  connectivity_macos: ^0.0.3
```

Comment:
connectivity_macos should be > 0.1.0 so it doesn't get penalized in pub.dev for having a very low version number.

Reply:
Also: is `flutter: ">=1.12.13+hotfix.5 <2.0.0"` the right constraint for federated macos plugins? When was federation for macos introduced?

Reply:
I think the ability to federate plugins was introduced in 1.10.0, so this should be ok